tests/test_sww.cpp

```diff
@@ -151,6 +151,13 @@ TEST( MeshSWWTest, Cairns )
   EXPECT_DOUBLE_EQ( 0, min );
   EXPECT_DOUBLE_EQ( 6.7305092811584473, max );
 
+  const char *referenceTime;
+  referenceTime = MDAL_G_referenceTime( g );
+  EXPECT_EQ( std::string( "none" ), std::string( referenceTime ) );
```

Comment:
i would return empty string in case format/dataset does not have reference time at all